README.md

```diff
@@ -7,7 +7,7 @@
 3. `cd functions`
 4. `npm install -g firebase-tools`
 5. `npm install`
-6. run `firebase login` then login with your betterSG email
+6. run `firebase login --no-localhost` then login with your betterSG email
```

Comment:
`firebase login` uses redirects to complete the auth, which gives issues in a remote development setup. `firebase login --no-localhost` allows you to copy a code and paste into the CLI, which works anywhere.